tests/test_srf.py

```diff
@@ -190,6 +200,15 @@ def test_darfield_srf():
         assert len(segment) == segment_header["nstk"] * segment_header["ndip"]
         assert (segment["dip"] == segment_header["dip"]).all()
         assert (segment["stk"] == segment_header["stk"]).all()
+    for (_, header), plane in zip(darfield_srf.header.iterrows(), darfield_srf.planes):
+        assert header[["elat", "elon"]].values == pytest.approx(plane.centroid[:2])
+        assert header["wid"] == pytest.approx(plane.width)
+        assert header["len"] == pytest.approx(plane.length)
+        assert header["len"] == pytest.approx(plane.length)
+        # the dip in the Darfield SRF makes no sense!
+        # assert header["dip"] == pytest.approx(plane.dip)
```

Comment:
Is this fine?

Reply:
The "dip" given in the SRF is > 90 degrees which makes no sense.